.github/workflows/unit_tests.yml

```diff
@@ -71,16 +72,16 @@ jobs:
           NAME: ${{ matrix.device }}
         run: |
           gem install coveralls-lcov
-          gem install cocoapods -v '1.9.3'
+          gem install cocoapods -v '1.11.3'
           pod repo update
           pod install
           HOMEBREW_NO_INSTALL_CLEANUP=true brew update && brew install jq
 
           # github actions are very poor in supporting iOS simulators:
           # - to find pre-installed xcode version, run this:
-          # > ls /Applications/
+          ##ls /Applications/
           # - to find supported simulator os versions, run this (and find simulator with non-error "datapath")
-          # > xcrun simctl list --json devices
+          ##xcrun simctl list --json devices
```

Comment:
are these commands no longer required?

Reply:
They're just for debugging to check available iOS simulators etc installed in github actions VM.